x-pack/plugins/siem/public/components/link_to/redirect_to_timelines.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+const TIMELINES_PAGE_NAME = 'timelines';
+
+export const RedirectToTimelinesPage = () => {
+  return <Redirect to={`/${TIMELINES_PAGE_NAME}`} />;
+};
```

Comment:
nit: This can be implicit return

Reply:
This is copy 🍝  from another `redirect_to_...` file in the same directory. I will update all files in the directory to use implicit return to avoid this in the future.